CONTRIBUTING.md

```diff
@@ -89,8 +78,6 @@ The default value for every parameter is `None`. Note that once a parameter valu
 
 `mdb_source_id` and `data_type` cannot be updated. All other parameters can.
 
-Note that you must pass an `api_key_parameter_value` if your source has `authentication_type = 1` or `authentication_type = 2` and that you want to update the direct download URL or authentication-related fields. The `api_key_parameter_value` will not be stored and is used only for testing before the source is updated in the database.
```

Comment:
Here as well, I would mention that these sources can't be updated from forks, if it makes sense from a user perspective.

Reply:
Just the source URLs, correct?

Reply:
Meaning that other fields could be updated on forks for sources requiring authentication? I would not be possible in current state. 

The problem is that we need to download a source when it's added or updated (see [here](https://github.com/MobilityData/mobility-database-catalogs/blob/6e7f50a1eabc38e1191de0b0db09586c893eef68/.github/workflows/direct_download_urls_test_for_sources.yml#L161)). If we update a field, say municipality, on a fork, we would still need to download the source, which would raise an exception in the `direct_download_urls_test_for_sources.yml` workflow. At the moment, we test updated sources without looking at field was updated - it's trickier to do. Maybe we changing the behavior would be useful if you see value in this use case.

Reply:
I think at a MVP I can just specify in the documentation that updates to API sources should be requested through the Google form.